yt/visualization/plot_container.py

```diff
@@ -226,6 +238,24 @@ def __init__(self, data_source, figure_size, fontsize):
         self._minorticks = {}
         self._field_transform = {}
 
+        self.setup_defaults()
+
+    def setup_defaults(self):
+        def default_from_config(key, default):
```

Comment:
you may want to use `functools.wraps` here

Reply:
What would be the benefit? `default_from_config` is not a decorator but is merely used to create a nice closure to the `getter` function.

Reply:
sorry I misread this